Mathlib/Data/Prod/Basic.lean

```diff
@@ -222,17 +222,19 @@ instance {r : α → α → Prop} {s : β → β → Prop} [IsStrictOrder α r]
     | (_, _), (_, _), .right _ _,     .left  _ _ hr₂ => (irrefl _ hr₂).elim
     | (_, _), (_, _), .right _ hs₁,   .right _ hs₂   => antisymm hs₁ hs₂ ▸ rfl⟩
 
-instance is_total_left {r : α → α → Prop} {s : β → β → Prop} [IsTotal α r] :
+instance isTotal_left {r : α → α → Prop} {s : β → β → Prop} [IsTotal α r] :
     IsTotal (α × β) (Lex r s) :=
   ⟨fun ⟨a₁, _⟩ ⟨a₂, _⟩ => (IsTotal.total a₁ a₂).imp (Lex.left _ _) (Lex.left _ _)⟩
+#align is_total_left isTotal_left
```

Comment:
Do we align instances?

Reply:
Maybe? I guess it's only _necessary_ when the names are used on the mathlib3 side. I'm not sure if we should
* always align named instances
* only do it after checking to see if mathlib3 uses the name
* never align named instances